From 4460c715bae90baba7b700d551888eb6f215565c Mon Sep 17 00:00:00 2001
From: Athenas Jimenez <athenas.jimenez.gonzalez@intel.com>
Date: Tue, 24 Sep 2019 01:19:37 +0000
Subject: [PATCH] Drop tag-single-build plugin

---
 build.gradle.kts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/build.gradle.kts b/build.gradle.kts
index 868815a6..ec1d1e89 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -32,7 +32,6 @@ plugins {
     // We have to apply it here at the moment, so that when the build scan plugin is auto-applied via --scan can detect that
     // the plugin has been already applied. For that the plugin has to be applied with the new plugin DSL syntax.
     com.gradle.`build-scan`
-    id("org.gradle.ci.tag-single-build") version("0.67")
 }
 
 defaultTasks("assemble")
-- 
2.23.0

