From 8ea8a19fc497f7a41c1709fd1761464aab23b129 Mon Sep 17 00:00:00 2001
From: Athenas Jimenez <athenas.jimenez.gonzalez@intel.com>
Date: Thu, 30 May 2019 16:30:06 +0000
Subject: [PATCH] Enable local fonts

---
 subprojects/docs/docs.gradle | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/subprojects/docs/docs.gradle b/subprojects/docs/docs.gradle
index e876b265..41a7c969 100755
--- a/subprojects/docs/docs.gradle
+++ b/subprojects/docs/docs.gradle
@@ -41,18 +41,16 @@ apply plugin: 'javascript-base'
 apply plugin: 'org.asciidoctor.convert'
 
 repositories {
-    javaScript.googleApis()
-
     ivy {
         name "Google Fonts"
-        url "http://themes.googleusercontent.com/static/fonts/"
+        url "/builddir/.m2/repository"
         layout 'pattern', {
             artifact '[organisation]/v[revision](/[classifier])(.[ext])'
-            ivy '[organisation]/v[revision]/ivy(.[ext])'
+            ivy '[organisation]/v[revision]/ivy-[module](.[ext])'
         }
     }
 
-    maven { url 'https://repo.gradle.org/gradle/gradle-build-internal' }
+    mavenLocal()
 }
 
 configurations {
-- 
2.21.0

